cloudflare/resource_cloudflare_load_balancer_monitor.go

```diff
@@ -135,20 +141,15 @@ func resourceCloudflareLoadBalancerPoolMonitorCreate(d *schema.ResourceData, met
 	client := meta.(*cloudflare.API)
 
 	loadBalancerMonitor := cloudflare.LoadBalancerMonitor{
-		ExpectedBody:  d.Get("expected_body").(string),
-		ExpectedCodes: d.Get("expected_codes").(string),
-		Method:        d.Get("method").(string),
-		Timeout:       d.Get("timeout").(int),
-		Type:          d.Get("type").(string),
-		Path:          d.Get("path").(string),
-		Interval:      d.Get("interval").(int),
-		Retries:       d.Get("retries").(int),
-	}
-
-	if header, ok := d.GetOk("header"); ok {
-		loadBalancerMonitor.Header = expandLoadBalancerMonitorHeader(header)
+		Timeout:  d.Get("timeout").(int),
+		Type:     d.Get("type").(string),
+		Interval: d.Get("interval").(int),
+		Retries:  d.Get("retries").(int),
 	}
 
+	//
```

Comment:
Similar comment to https://github.com/terraform-providers/terraform-provider-cloudflare/pull/428/files#r307064855

Reply:
removed